app.json

```diff
@@ -1,4 +1,4 @@
 {
-  "name": "How Long To Beat",
-  "displayName": "How Long To Beat"
-}
+  "name": "hltb",
+  "displayName": "hltb"
```

Comment:
Why did we had to change this?

Reply:
I think react changed this when I ran the project.
I will revert it because I suppose it does not need to be changed.